kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -126,15 +126,6 @@
           />
         </div>
       </div>
-
```

Comment:
This seems like important functionality?

Reply:
It sure is - yipes. In my mind I conflated this and vs. the progress bar which is being updated in #8112, but they are (obviously) different. Will fix